### PR TITLE
[CPDLP-3812] Remove NPQ admin logic (NPQ Post Separation Cleanup)

### PR DESCRIPTION
**Pull request by leandroalemao**
### Context

- Ticket: [CPDLP-3812](https://dfedigital.atlassian.net/browse/CPDLP-3812)

Now we’ve turned on the feature flag in ECF to hide NPQ logic we can remove any admin logic where we used to show NPQ functionality.

### Changes proposed in this pull request

Remove any NPQ logic from ECF admin:
- Any hidden tabs/services and tools;
- Anywhere the feature flag is hiding functionality;
- Only limit this to the admin, finance is in a different ticket;

[CPDLP-3812]: https://dfedigital.atlassian.net/browse/CPDLP-3812?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ethax-ross — COMMENTED** [review]
Looks good! there's a bunch of these in the admin controllers - is it worth removing the reference to NPQ here? might even be worth moving it to the base admin controller while we're at it as its duplicated a bunch of times...

```
# Get the school from the induction record for ECTs and from the participant profile for NPQs
def school
  @school ||= @participant_profile.latest_induction_record&.school || 
  @participant_profile.school
end
```

There's also this view still kicking about:

```
app/views/admin/participant/details/_show_npq.html.erb
```

And a few references to NPQ in:

```
app/views/admin/participant/details/show.html.erb
```

**Comment by leandroalemao** [comment]
> Looks good! there's a bunch of these in the admin controllers - is it worth removing the reference to NPQ here? might even be worth moving it to the base admin controller while we're at it as its duplicated a bunch of times...
> 
> ```
> # Get the school from the induction record for ECTs and from the participant profile for NPQs
> def school
>   @school ||= @participant_profile.latest_induction_record&.school || 
>   @participant_profile.school
> end
> ```
> 
> There's also this view still kicking about:
> 
> ```
> app/views/admin/participant/details/_show_npq.html.erb
> ```
> 
> And a few references to NPQ in:
> 
> ```
> app/views/admin/participant/details/show.html.erb
> ```

Tks @ethax-ross 
Addressed in https://github.com/DFE-Digital/early-careers-framework/pull/5358/commits/b79194e558d19ac0e032730008127137e04cee06 👍

**Review by ethax-ross — APPROVED** [review]

File: spec/services/admin/participants/search_spec.rb [hunk 1]
```diff
@@ -141,31 +139,6 @@
...
-        let(:results) { search.call(ParticipantProfile, search_term:) }
-
-        it "returns matching participants" do
-          expect(results).to include(pp_1)
-        end
-
-        it "doesn't return non-matching participants" do
-          expect(results).not_to include(pp_2, pp_3)
-        end
-      end
```
Comment (mooktakim) [inline spec/services/admin/participants/search_spec.rb:157]: I think we should keep this test, but change it for `.or(TeacherProfile.trn_matches(search_term))`

Also on `Admin::Participants::Search `, we can remove:
```ruby
        .or(NPQApplication.where(teacher_reference_number: search_term))
        .or(NPQApplication.where(id: search_term))
```

**Review by cwrw — COMMENTED** [review]
looks good @leandroalemao to me, just @mooktakim's comment otherwise 👍

File: spec/services/admin/participants/search_spec.rb (hunk 1 above)
Comment (leandroalemao) [reply spec/services/admin/participants/search_spec.rb:157]: Hi @mooktakim Addressed in [3a38988](https://github.com/DFE-Digital/early-careers-framework/pull/5358/commits/3a3898865a874725f0babbc4cfc88d72e9b9d2b2). Tksss. Also I noticed there's already another spec in the same file testing the search by TRN..

**Review by mooktakim — APPROVED** [review]

**Review by cwrw — APPROVED** [review]

**Pull request merged**